privacyidea/api/lib/prepolicy.py

```diff
@@ -711,6 +714,30 @@ def check_max_token_user(request=None, action=None):
             if already_assigned_tokens >= max_value:
                 g.audit_object.add_policy(limit_list.get(str(max_value)))
                 raise PolicyError(ERROR)
+
+        # check maximum active tokens of user
+        limit_list = policy_object.get_action_values(ACTION.MAXACTIVETOKENUSER,
+                                                     scope=SCOPE.ENROLL,
+                                                     realm=user_object.realm,
+                                                     resolver=user_object.resolver,
+                                                     user=user_object.login,
+                                                     client=g.client_ip)
+        if limit_list:
+            # we need to check how many active tokens the user already has assigned!
+            tokenobject_list = get_tokens(user=user_object, active=True)
+            if serial:
+                for tok in tokenobject_list:
+                    if tok.token.serial == serial and tok.is_active():
```

Comment:
I think ``tok.is_active()`` should always be true, because we pass ``active=True`` to ``get_tokens`` above.

Reply:
Will do

Reply:
Done in 3ad5a51d